tests/requirements.txt

```diff
@@ -14,6 +14,7 @@ tensorboard
 ipython
 ipywidgets
 pandas
+pyarrow
```

Comment:
As a future reference: This is required as pandas started to emit DeprecationWarning when pyarrow is missing.
https://ci.preferred.jp/pytorch-pfn-extras.torch111-win/154993/#L631

(thanks @linshokaku for this info!)